test/gtest/uct/test_tag.cc

```diff
@@ -14,6 +14,14 @@ extern "C" {
     _UCT_INSTANTIATE_TEST_CASE(_test_case, rc_mlx5) \
     _UCT_INSTANTIATE_TEST_CASE(_test_case, dc_mlx5)
 
+
+/* proxy function for uct_iface_release_desc to eliminate
+ * errors related to failed of inlining uct_iface_release_desc */
```

Comment:
Wrapper for uct_iface_release_desc() to eliminate errors of failing to inline it when passed to std::for_each
BTW, can we use lambda expression in std::for_each instead of named wrapper?

Reply:
> BTW, can we use lambda expression in std::for_each instead of named wrapper?

as I remember we have minimal supported C++ standard C11, I believe we can use lambdas (not sure if GCC 4.4 will compile it)